fms/models/llama.py

```diff
@@ -430,7 +432,9 @@ def _rename_weights_to_fms(orig_sd):
             new_name = re.sub(pattern, repl, new_name)
         new_sd[new_name] = param
 
-    return new_sd
+    fused_sd = _convert_fused_qkv_pre_0_0_6(new_sd)
```

Comment:
this makes the function name look a bit awkward. you're using it here because meta weights happen to be provided un-fused, and that's not related to the particular fms version

Reply:
Might make sense to just change this to `_convert_fused_qkv`. The adapter for fms pre 0.0.6 already has that information in the registration.

Reply:
though let's make sure it's clear which direction it goes (i.e. `_fuse_qkv` or "_convert_to_fused_qkv`) otherwise it's not immediately obvious if the function converts from or to fused.

Reply:
Fixed in latest commit